src/microbe_stage/SpawnSystem.cs

```diff
@@ -350,10 +303,10 @@ private int DespawnEntities(Vector3 playerPosition)
             }
 
             var entityPosition = ((Spatial)entity).Translation;
-            var squaredDistance = (playerPosition - entityPosition).LengthSquared();
+            var distance = (playerPosition - entityPosition).Length();
```

Comment:
LengthSquared should absolutely be kept here as Length is *much* more expensive to compute as it requires as square root computation.

Reply:
Oh I remember what I was thinking. I changed spawning so it would use distance without ever needing to call Length or a sqrt function. But then I didn't think about despawning when I changed it... 

I just fixed that silly mistake.